### PR TITLE
Enhancement/add top temp plugin

### PR DESCRIPTION
**Pull request by eliasbakken**

### CODE REVIEW

File: roles/install_octoprint/files/config_recore.yaml [hunk 1]
```diff
@@ -0,0 +1,118 @@
+appearance:
+  name: Recore
+folder:
+  uploads: /usr/share/models
```
Comment (goeland86) [inline roles/install_octoprint/files/config_recore.yaml:4]: This is non-standard and messes with the file permissions.
Normally the path should be `/home/debian/.octoprint/uploads/` instead.

**Review by goeland86 — CHANGES_REQUESTED** [review]
The uploads directory in OctoPrint is no longer `/usr/share/models`, or at least it shouldn't be.
It can cause issues with file/folder permissions, and gcode files are user-space files. It makes no sense to have them in `/usr/`

**Review by goeland86 — APPROVED** [review]

**Comment by goeland86** [comment]
Why are we still using `/usr/share/models`?

**Comment by eliasbakken** [comment]
It was the location chosen years ago. I remember choosing it. It's been like that probably for 5-6 years. If we write an issue for it, we can change it, but it must be done separately from this PR.

**Comment by goeland86** [comment]
I remember changing it in the first refactor versions though. I'll make an issue for it.

**Pull request merged**